juno/assetlist.json

```diff
@@ -1307,6 +1307,50 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/void.png"
       }      
+    },
+    {
+      "description": "Slica",
+      "type_asset": "cw20",
+      "address": "juno10vgf2u03ufcf25tspgn05l7j3tfg0j63ljgpffy98t697m5r5hmqaw95ux",
+      "denom_units": [
+        {
+          "denom": "cw20:juno10vgf2u03ufcf25tspgn05l7j3tfg0j63ljgpffy98t697m5r5hmqaw95ux",
+          "exponent": 0
+        },
+        {
+          "denom": "slica",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:juno10vgf2u03ufcf25tspgn05l7j3tfg0j63ljgpffy98t697m5r5hmqaw95ux",
+      "name": "Slica",
+      "display": "slica",
+      "symbol": "SLCA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/slica.png"
+      }      
+    },
+    {
+      "description": "Pepec",
+      "type_asset": "cw20",
+      "address": "juno10vgf2u03ufcf25tspgn05l7j3tfg0j63ljgpffy98t697m5r5hmqaw95ux",
+      "denom_units": [
+        {
+          "denom": "cw20:juno10vgf2u03ufcf25tspgn05l7j3tfg0j63ljgpffy98t697m5r5hmqaw95ux",
+          "exponent": 0
+        },
+        {
+          "denom": "pepec",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:juno10vgf2u03ufcf25tspgn05l7j3tfg0j63ljgpffy98t697m5r5hmqaw95ux",
```

Comment:
as mentioned before, these two tokens can't have the same address and base denom. Please correct. I think PEPE may be this: [juno1epxnvge53c4hkcmqzlxryw5fp7eae2utyk6ehjcfpwajwp48km3sgxsh9k](https://www.mintscan.io/juno/wasm/contract/juno1epxnvge53c4hkcmqzlxryw5fp7eae2utyk6ehjcfpwajwp48km3sgxsh9k)